### ISSUE TITLE
InputTags - focusing component with Tab works as not expected

### ISSUE THREAD

**Issue by mihalichzh** [semrush/intergalactic#709]
**Describe the bug**
During a11y testing for one of our products, I noticed that focusing InputTag component with the keyboard doesn't work as expected (see att. video).

**To Reproduce**
1. Attempt to focus InputTag component using keyboard

**Expected behavior**
Input focus behavior is straightforward (for instance, similar to [this](https://github.com/accessible-components/tag-input) component - not sure if its behavior 100% correct from a11y point of view, but it works more convenient at least).

**Actual behavior**
When you attempt to focus on InputTag component, on the first Tab press, default focus ring for 'data-ui-name="InputTags"' component is shown. The custom focus ring appears only after the second Tab press, but 'data-ui-name="ScrollArea.Container"' node inside the InputTags component is focused. See att. video for better understanding.

https://github.com/semrush/intergalactic/assets/39348763/7e80e9c3-da90-4883-b9d7-611dda8342a1

**Comment by msereniti** [comment]
Issue was fixed in the latest release
https://github.com/semrush/intergalactic/releases/tag/v15.7.0
